templates/expo-template-bare-minimum/ios/Podfile

```
@@ -5,6 +5,9 @@ require File.join(File.dirname(`node --print "require.resolve('@react-native-com
 require 'json'
 podfile_properties = JSON.parse(File.read(File.join(__dir__, 'Podfile.properties.json'))) rescue {}
 
+# Libraries use this env flag to determine whether to add React-RCTFabric as a dependency.
+ENV['RCT_NEW_ARCH_ENABLED'] = podfile_properties['expo.newArchEnabled'] ? '1' : '0'
```

```suggestion
ENV['RCT_NEW_ARCH_ENABLED'] = podfile_properties['newArchEnabled'] ? '1' : '0'
```

let's use `newArchEnabled` to align with android key.

---

updated